lib/install/management/appliance.go

```diff
@@ -394,14 +398,8 @@ func (d *Dispatcher) reconfigureApplianceSpec(vm *vm.VirtualMachine, conf *metad
 	spec.ExtraConfig = d.getPresetExtraconfig(conf)
 
 	cfg := make(map[string]string)
-	extraconfig.EncodeWithPrefix(extraconfig.MapSink(cfg), conf, "guestinfo.vch")
+	extraconfig.Encode(extraconfig.MapSink(cfg), conf)
 	spec.ExtraConfig = append(spec.ExtraConfig, extraconfig.OptionValueFromMap(cfg)...)
```

Comment:
It's not duplicated, but it is set twice - the only reason it's in the initial spec for the applianceVM creation is so that the VM can be identified as a VCH appliance even if it's never powered on.
I'll add a comment.